# MoveIt-specific simulation settings
moveit_sim_hw_interface:
  joint_model_group: controllers_initial_group_
  joint_model_group_pose: controllers_initial_pose_
# Settings for ros_control control loop
generic_hw_control_loop:
  loop_hz: 300
  cycle_time_error_threshold: 0.01
# Settings for ros_control hardware interface
hardware_interface:
  joints:
    - a_bot_shoulder_pan_joint
    - a_bot_shoulder_lift_joint
    - a_bot_elbow_joint
    - a_bot_wrist_1_joint
    - a_bot_wrist_2_joint
    - a_bot_wrist_3_joint
    - b_bot_shoulder_pan_joint
    - b_bot_shoulder_lift_joint
    - b_bot_elbow_joint
    - b_bot_wrist_1_joint
    - b_bot_wrist_2_joint
    - b_bot_wrist_3_joint
  sim_control_mode: 1  # 0: position, 1: velocity
# Publish all joint states
# Creates the /joint_states topic necessary in ROS
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50
controller_list:
  []